version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "3.20.0-SNAPSHOT"
```

Comment:
nah, I just didn't know what it did, I'd be happy if you stuck that into the patch–we'll have to change a small amount of tooling, but you're right that it will make our lives easier.  if you don't feel like putting it back in, not a big deal though.